### DISCUSSION TITLE
Grlx Sprout - Support Windows Management

### DISCUSSION THREAD

**Discussion by willchenmark** [gogrlx/grlx#113] [Ideas]
It would be great to see support for Sprouts on Windows systems, at least to the point where the sprout can be run on a Windows system and then used to execute commands on remotely from the Farmer.

Functionally, if we had these two capabilities it would allow admins to begin using Grlx to manage Windows systems, even if it is a less pleasant experience than working with Linux based sprouts.
- Access to CMD/Powershell for one off commands and scripts
- File handling

It would be nice to see modules to support these functionalities as well.
- Service handling
- Local GPO configuration
- Registry configuration
- User management
- Driver management
- Software management (through chocolatey or something else potentially?)

Just having access to Powershell and file management capabilities would allow an admin to deploy the functionality that the suggested modules would provide through scripts and one off commands. And the Powershell module would likely form a basis for the creation of these additional suggested modules.

**Comment by taigrr** [comment]
Thanks for your interest in grlx. We currently have no plans to support Windows at this time, but I'll leave the discussion open, and if there's substantive desire from users, we'll consider adding it to our roadmap.
